### PR TITLE
[chains] Expose RPC timeout

### PR DESCRIPTION
**Pull request by marius-baseten**

### CODE REVIEW

File: docs/chains/doc_gen/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@ This generation process of the documentation is *extremely* scrappy and just
 an interim solution. It requires significant manual oversight and the code
 quality in this directory is non-existent.
 
+Extra deps required:
```
Comment (bolasim) [inline docs/chains/doc_gen/README.md:5]: is this intentional?

File: truss-chains/truss_chains/definitions.py [hunk 2]
```diff
@@ -365,10 +365,13 @@ def get_asset_spec(self) -> AssetSpec:
         return self.assets.get_spec()
 
 
+DEFAULT_TIMEOUT_SEC = 600
```
Comment (bolasim) [inline truss-chains/truss_chains/definitions.py:368]: this seems extremely high to me...

**Review by bolasim — APPROVED** [review]

File: docs/chains/doc_gen/README.md (hunk 1 above)
Comment (marius-baseten) [reply docs/chains/doc_gen/README.md:5]: Yes.

File: truss-chains/truss_chains/definitions.py (hunk 2 above)
Comment (marius-baseten) [reply truss-chains/truss_chains/definitions.py:368]: True. This unfortunately factored in the "wake time" for a nested chain. We have improved chain wake up on the radar, but it's currently not there yet. Do you think it's safe to reduce the default, without causing unintended side effects for existing use cases at the moment?

**Pull request merged**